docs/supported-libraries.md

```diff
@@ -119,7 +119,7 @@ These are the supported libraries and frameworks:
 | [Spark Web Framework](https://github.com/perwendel/spark)                                                                                   | 2.3+                          | N/A                                                                                                                                                                                                                                                                                                                                                                                     | Provides `http.route` [2]                                                                                                               |
 | [Spring Batch](https://spring.io/projects/spring-batch)                                                                                     | 3.0+ (not including 5.0+ yet) | N/A                                                                                                                                                                                                                                                                                                                                                                                     | none                                                                                                                                    |
-| [Spring Boot](https://spring.io/projects/spring-boot)                                                                                       |                               | [opentelemetry-spring-boot-resources](../instrumentation/spring/spring-boot-resources/library)                                                                                                                                                                                                                                                                                          | none                                                                                                                                    |
+| [Spring Boot](https://spring.io/projects/spring-boot)                                                                                       |                               | [opentelemetry-spring-boot-resources](../instrumentation/spring/spring-boot-resources/javaagent)                                                                                                                                                                                                                                                                                        | none                                                                                                                                    |
```

Comment:
the 3rd column is only for library instrumentation

do we have other Spring Boot library instrumentation we want to link here? maybe the starter itself?